### PR TITLE
Support changing document title

### PR DESCRIPTION
**Pull request by ducfilan**
```
I want to change the Document title on a tab when ignoring rooms.
Example: 
There are 1 unread message from room id 1, web title: [1](1)Chatwork - My Chat.
I ignored room id 2, when a new message comes from room id 2, the title changed to [2]Chatwork - My Chat. It should be [1](1) - RoomX

So i added that functionality.
```

### CODE REVIEW

**Comment by chocoby** [comment]
Ah, sorry! I will review this.

**Comment by chocoby** [comment]
Thanks! It seems good.

I think that seperated room names (`room a|room b`) should be an option.
I will try it and release later.

Btw, are you using this extension?

**Comment by ducfilan** [comment]
@chocoby Yes I am using it in my company to ignore some public rooms and get notified when being mentioned only (TO).

**Comment by chocoby** [comment]
Wow! Thank you! :smile:

File: src/coffee/contentscript.coffee [hunk 1]
```diff
@@ -1,31 +1,50 @@
...
-  timer = setTimeout ->
-    $.each excludeRooms, (_, roomId) ->
-      selector = "#_roomListItems li[data-rid=#{roomId}]"
+getMentionCounts = ->
+  $('#_roomListArea .mention').length
 
+getUnreadRoomsName = ->
+  l_unreadRoomsName = $.map($('#_roomListItems .roomUnread .chatListTitleArea'), $.text)
+  if l_unreadRoomsName.length == 0
+    l_unreadRoomsName = [ 'No message!' ]
```
Comment (chocoby) [inline src/coffee/contentscript.coffee:11]: @ducfilan 

I think that the following title is displayed in this code if we have no message.

```
[1](0) - No message!
```

I would like to change to the following title.

```
[0] - ChatWork
or
[0] - No message!
```

What do you think?

**Comment by chocoby** [comment]
v1.0.0.rc1 is released.
https://github.com/chocoby/chatwork-mute-room-extension/releases/tag/v1.0.0.rc1

File: src/coffee/contentscript.coffee (hunk 1 above)
Comment (ducfilan) [reply src/coffee/contentscript.coffee:11]: @chocoby Yes, I think your change's good :+1:

**Comment by chocoby** [comment]
Published to the Chrome Web Store
https://chrome.google.com/webstore/detail/chatwork-mute-room/hnilammhfaleikbmmobhedpohmkbonab

**Pull request merged**